### DISCUSSION TITLE
Why is Propagation removed?

### DISCUSSION THREAD

**Discussion by h725rk** [ZoeyVid/NPMplus#1273] [Q&A] [answered]
Hi,

why is the propagation time by DNS challange removed?
My DNS-Provider is Netcup. It takes until 15 Minutes to propagate the TXT-DNS entries.
Can you put it back or is there an other workaround?

**Comment by Zoey2936** [comment]
Hello, I will readd this option, but it may take some time

**Reply by h725rk** [reply]
Thank you. I will wait.

**Reply by h725rk** [reply]
Do you have schedule for this option?

**Reply by Zoey2936** [reply]
now I know what I wanted to do, but couldn't find the discussion anymore. Yes will work on it soon. I unmarked it as answer yesterday so it is visible for me

**Reply by Zoey2936** [reply, marked as answer]
interesting, I readded it already some time ago, you should already be able to use it again

**Comment by h725rk** [comment]
I have look at the changelog, but I don't find nothing about it. 
I tested it and its there. Perfect
Vielen Dank / Thank you very much

**Reply by Zoey2936** [reply]
I added it here again: https://github.com/ZoeyVid/NPMplus/releases/tag/2024-12-14-r1

**Reply by h725rk** [reply]
Damn, I have overlook it.
